lib/src/buttons/toolbar/toolbar_pulldown_button.dart

```diff
@@ -0,0 +1,124 @@
+  ///
+  /// For this reason, you must set the [label] property of all
+  /// [MacosPulldownMenuItem]s, as it's necessary for setting the submenu's
+  /// content.
+  final List<MacosPulldownMenuEntry>? items;
+
+  @override
+  Widget build(BuildContext context, ToolbarItemDisplayMode displayMode) {
+    final brightness = MacosTheme.of(context).brightness;
+
+    if (displayMode == ToolbarItemDisplayMode.inToolbar) {
+      return Padding(
+        padding: const EdgeInsets.symmetric(vertical: 6.0),
+        child: MacosPulldownButtonTheme(
+          data: MacosPulldownButtonTheme.of(context).copyWith(
+            iconColor: brightness.resolve(
+              const Color.fromRGBO(0, 0, 0, 0.5),
+              const Color.fromRGBO(255, 255, 255, 0.5),
+            ),
+          ),
+          child: MacosPulldownButton(
+            icon: icon,
+            items: items,
+          ),
+        ),
+      );
+    } else {
+      // We should show a submenu for the pulldown button items.
+      final subMenuKey = GlobalKey<ToolbarPopupState>();
+      List<ToolbarOverflowMenuItem> subMenuItems = [];
+      bool isSelected = false;
+      // Convert the original pulldown menu items to toolbar overflow menu items.
+      items?.forEach((element) {
+        if (element is MacosPulldownMenuItem) {
+          assert(element.label != null,
+              'When you use a MacosPulldownButton in the Toolbar, you must set the label property for all MacosPulldownMenuItems.');
+          subMenuItems.add(
+            ToolbarOverflowMenuItem(
+              label: element.label!,
+              onPressed: () {
+                element.onTap?.call();
+                // Close the initial overflow menu as well.
+                Navigator.pop(context);
+              },
+            ),
+          );
+        }
+      });
+      return StatefulBuilder(
+        builder: (context, setState) {
+          return ToolbarPopup(
+            key: subMenuKey,
+            content: (context) => MouseRegion(
+              child: ToolbarOverflowMenu(children: subMenuItems),
+              onExit: (e) {
+                // Moving the mouse cursor outside of the submenu should
+                // dismiss it.
+                subMenuKey.currentState?.removeToolbarPopupRoute();
+                setState(() => isSelected = false);
+              },
+            ),
+            verticalOffset: 0.0,
+            horizontalOffset: 0.0,
+            position: ToolbarPopupPosition.side,
+            placement: ToolbarPopupPlacement.start,
+            child: MouseRegion(
+              onHover: (e) {
+                subMenuKey.currentState
+                    ?.openPopup()
+                    .then((value) => setState(() => isSelected = false));
+                setState(() => isSelected = true);
+              },
+              child: ToolbarOverflowMenuItem(
+                label: label,
+                subMenuItems: subMenuItems,
+                onPressed: () {},
```

Comment:
Why is this `onPressed` not set to anything?

Reply:
This is for when the pulldown button is converted to an entry for the overflowed menu, which opens a submenu on hover (and not on click). If set to null, it would show up as disabled.